src/library/coap_test.cpp

```diff
@@ -344,6 +344,8 @@ class MockEndpoint : public Endpoint
 
 TEST_CASE("coap-message-confirmable", "[coap]")
 {
+    bool send_separate_empty_ack = false;
```

Comment:
```suggestion
    bool sendSeparateEmptyAck = false;
```
since we do we migrate to `lowercase_separated_by_underscores` style.